RedHatInsights/vulnerability-ui
# refactor: Remove the last insights.chrome call

Fixes https://issues.redhat.com/browse/RHIF-213.

This removes the last occurrence of direct call to insights.chrome. Together with this change, CVEs component is refactored to abstract the RBAC logic from the core component and make it cypress-testable.

## How to test

- Component tests must pass.
- All the views should have no visible change. The permissions must be respected the same as before this change.
